apps/coscrad-frontend/src/components/header/header.tsx

```diff
@@ -7,7 +7,7 @@ import { ConfigurableContentContext } from '../../configurable-front-matter/conf
 import { NavMenuContainer } from '../nav-menu/nav-menu-container';
 
 const StyledAppBar = styled(AppBar)({
-    backgroundColor: '#2cb5af',
+    backgroundColor: 'primary.main',
     position: 'sticky',
```

Comment:
Fallback if a color isn't defined in the content.config?

Reply:
```
             primary: {
                    main: theme.primaryColor || blue[500],
                },
```
                
                something like this? @aaron-plahn

Reply:
@aaron-plahn is this a solution for fallback if a color isn't defined in the content.config?

Reply:
I don't think you need this. MUI should already set defaults.